tests/util/mod.rs

```diff
@@ -0,0 +1,50 @@
+use std::sync::{Arc, Once, ONCE_INIT};
+
+use error_chain::ChainedError;
+use jni::{InitArgsBuilder, JNIEnv, JNIVersion, JavaVM};
+use jni::errors::Result;
+
+
+pub fn jvm() -> &'static Arc<JavaVM> {
```

Comment:
Looks cool that individual ITs won't have to duplicate that code.

Reply:
I wonder if this guy may be easily migrated to `jvm`: https://github.com/prevoty/jni-rs/blob/master/tests/java_integers.rs

Reply:
Honestly, I just did not want to go beyond the "topic" of the PR. 
But it seems really better to immediately make everything uniform.

Reply:
I see, that's reasonable for there are various views on PR and commit atomicity. I generally prefer to perform a system-wide refactoring when introducing a new better API that replaces duplicated code. It makes it likely you won't forget to do it, and other developers will use this single API (they copy-paste code, and might copy-paste the old version of it).